docs/source/concepts.rst

```diff
@@ -1,2 +1,78 @@
 Key Concepts
 ============
+
+Let's first recap our example reporting workflow:
```

Comment:
If going in order, they'll have just come from the example use case section—I find it pretty redundant to duplicate the image here. I can put some text linking back to that section at the top?